### ISSUE TITLE
Worked for 2-3 days and crash

### ISSUE THREAD

**Issue by renat2985** [oven-sh/bun#11742]
### Relevant log output

```shell
Bun v1.1.12 (43f0913c) Linux x64
Args: "bun" "socket.js"
Features: jsc http_server 
Builtins: "bun:main" "bun:sqlite" 
Elapsed: 260244586ms | User: 290250ms | Sys: 932927ms
RSS: 1.07GB | Peak: 128.82MB | Commit: 1.07GB | Faults: 0

panic(main thread): Segmentation fault at address 0x2
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

To send a redacted crash report to Bun's team,
please file a GitHub issue using the link below:
```

### Stack Trace (bun.report)

Bun v1.1.12 ([`43f0913`](<https://github.com/oven-sh/bun/tree/43f0913c3875cbf260c60868268adc5953b09a80>)) on linux x86_64 [AutoCommand]

Segmentation fault at address 0x00000002

- [`ld-temp.o:0`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80/ld-temp.o#L0>): `WTF::Unicode::convertReplacingInvalidSequences`
- `??`
- [`ld-temp.o:0`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80/ld-temp.o#L0>): `llint_relativePCBase`
- [`ld-temp.o:0`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80/ld-temp.o#L0>): `JSC::Interpreter::executeCall`
- [`ld-temp.o:0`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80/ld-temp.o#L0>): `JSC::ClonedArguments::getOwnPropertySlot`
- [`JSCJSValueInlines.h:0`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80//build/bun/bun-webkit/include/JavaScriptCore/JSCJSValueInlines.h#L0>): `JSC__JSObject__getIndex`
- [`array_list.zig:859`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80/src/deps/zig/lib/std/array_list.zig#L859>): `src.bun.js.bindings.bindings.JSGlobalObject.throw__anon_415889`
- [`hashtable_policy.h:433`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80//usr/bin/../lib/gcc/x86_64-linux-gnu/10/../../../../include/c++/10/bits/hashtable_policy.h#L433>): `uWS::TopicTree<uWS::TopicTreeMessage, uWS::TopicTreeBigMessage>::subscribe`
- [`server.zig:4095`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80/src/bun.js/api/server.zig#L4095>): `src.deps.uws.WebSocketBehavior.Wrap`
- [`basic_string.tcc:0`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80//usr/bin/../lib/gcc/x86_64-linux-gnu/10/../../../../include/c++/10/bits/basic_string.tcc#L0>): `uWS::InflationStream::inflate`

**Comment by Jarred-Sumner** [comment]
There are a few different things this could've been. Judging by the stack trace, it looks like either:
1. the computer might've run out of memory
2. the string passed to the  .subscribe() method was prematurely garbage collected
3. an exception was already thrown while attempting to throw another exception. 

#11743 covers 2 and 3

**Comment by Jarred-Sumner** [comment]
This should be fixed as of Bun v1.1.13. Please let us know if you still run into this crash.
